docs/src/reference/asciidoc/appendix/release-notes/8.1.0.adoc

```diff
@@ -0,0 +1,33 @@
+[[eshadoop-8.1.0]]
+== Elasticsearch for Apache Hadoop version 8.1.0
+
+coming::[8.1.0]
+
+[[new-8.1.0]]
+[float]
+=== Enhancements
+
+Core::
+- Support Scala 2.13 for Apache Spark 3.2.0
+https://github.com/elastic/elasticsearch-hadoop/pull/1841[#1841]
```

Comment:
I hadn't properly labeled 3 of these. 1841, 1839, and 1838 are all Spark-specific (rather than Core).

Reply:
Thanks. I'll get those relabelled here.